recipe/meta.yaml

```diff
@@ -7,11 +7,12 @@ package:
   version: {{ version }}
 
 source:
-  url: https://github.com/OpenFreeEnergy/alchemiscale/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: 4804dc95f53226553ca25716e51a99ce46e0ba74033f15316f8a00549f9782c1
+  git_url: https://github.com/OpenFreeEnergy/{{ name }}.git
+  git_rev: v{{ version }}
+  git_depth: -1
```

Comment:
What does `-1` do here?

Reply:
It pulls the full depth, which makes sure we get all the tags we need (the default is shallow so if we had too many commits to main before we run this (which can be a problem when we do a new build or migration 3 months after a release))